packages/remix-netlify-edge/server.ts

```diff
@@ -0,0 +1,58 @@
+import type { ServerBuild } from "https://esm.sh/@remix-run/server-runtime@1.3.5?pin=v77";
+import { createRequestHandler as createRemixRequestHandler } from "https://esm.sh/@remix-run/server-runtime@1.3.5?pin=v77";
```

Comment:
These are referencing older versions.

We should at least update to latest version

```suggestion
import type { ServerBuild } from "https://esm.sh/@remix-run/server-runtime@1.4.1?pin=v77";
import { createRequestHandler as createRemixRequestHandler } from "https://esm.sh/@remix-run/server-runtime@1.4.1?pin=v77";
```

Or remove version number all together

```suggestion
import type { ServerBuild } from "https://esm.sh/@remix-run/server-runtime?pin=v77";
import { createRequestHandler as createRemixRequestHandler } from "https://esm.sh/@remix-run/server-runtime?pin=v77";
```